### DISCUSSION TITLE
Passwordless Login with Login Service

### DISCUSSION THREAD

**Discussion by batuhanbilginn** [directus/directus#9492] [Archive]
Hey guys, I've created a custom endpoint to log in my users with NextAuth and I was using regular Login Service without password with "default" provider. As far as I know, we are not able to use this service without a password anymore. Is there a workaround for this problem? Thanks a lot!

**Comment by aidenfoxx** [comment]
There is no current workaround. The previous flow where you could authenticate without a password was a workaround for OAuth flows which don't use passwords. Now OAuth has its own dedicated flow, the workaround was removed.

There was a user looking into a better solution for generating access tokens here: #9284
